src/ResourceManager/Profile/Commands.Profile.Test/LoginCmdletTests.cs

```diff
@@ -41,6 +44,41 @@ public LoginCmdletTests(ITestOutputHelper output)
         }
 
         [Fact]
+        [Trait(Category.RunType, Category.CheckIn)]
```

Comment:
If you make the first trait Category.AcceptanceType, this will run as part of check-in